src/app/add-item/add-item.component.html

```diff
@@ -0,0 +1,13 @@
+<section class="add-item-section">
+    <div class="add-item-content">
+        <input type="text" class="add-textbox add-title" placeholder="Add the best title ever!" [(ngModel)]="itemTitle"/>
+        <input type="text" class="add-textbox add-description" placeholder="Add a description for the best todo item!" [(ngModel)]="itemDescription"/>
+        <input type="button" class="add-button" value="+" (click)="addItem()"/>
+    </div>
+</section>
+
+<section class="items-content">
+    <div class="item-content" *ngIf="items.length > 0">
+        <app-todo-item *ngFor="let item of items" [items]= "item"></app-todo-item>
+    </div>
+</section>
```

Comment:
This section does not add, only shows.
It should be outside this component (`add-item.component`)